### ISSUE TITLE
Create Issue with mandatory duedate

### ISSUE THREAD

**Issue by myki57** [lesstif/php-jira-rest-client#162]
Hello, 
Thanks for your fantastic work. Your API is really simple to use and so much powerfull to automate some process. 

I just have a question, I'm would like to create an issue but on some JIRA projects the duedate is mandatory. But I don't find where I can specify a due date when using IssueField method. 

Is this something that's not possible or I'm missing someting ?

Thanks in advance for your answer.

**Comment by lesstif** [comment]
hi @myki57 

i was forgot to implement setter method for duedate field and it's fixed [v1.23.1.](https://github.com/lesstif/php-jira-rest-client/releases/tag/1.23.1)

you can to set issue' duedate by _setDueDate_() method.

```php
$issueField->setProjectKey("TEST")
                ->setSummary("something's wrong")
                ->setAssigneeName("lesstif")
                ->setPriorityName("Critical")
                ->setIssueType("Bug")
                ->setDescription("Full description for issue")
                ->addVersion(["1.0.1", "1.0.3"])
                ->addComponents(['Component-1', 'Component-2'])
                // set issue security if you need.
                ->setSecurityId(10001 /* security scheme id */)
                ->setDueDate('2019-06-19')
            ;
```

thanks!

**Comment by myki57** [comment]
Thanks to you lesstif. 

And again great job you've done with this dev.
